src/transformStateWithClones.js

```diff
@@ -8,6 +8,31 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+
+  const previousVerOfStateArray = [];
+  let copyState = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(copyState, action.extraData);
+        break;
+      case 'removeProperties':
+        for (const keyToRemove of action.keysToRemove) {
+          if (copyState.hasOwnProperty(keyToRemove)) {
+            delete copyState[keyToRemove];
+          }
```

Comment:
```suggestion
          delete copyState[keyToRemove];
```
Можно без if, если такого свойства нет - просто ничего не произойдет

Reply:
Понял. Спасибо.